lua/caramba/multifile.lua

```
@@ -388,55 +388,104 @@ end
+    local parser = vim.treesitter.get_string_parser(content, lang)
+    local tree = parser:parse()[1]
+    if not tree then goto continue end
+
+    local root = tree:root()
+    local query = vim.treesitter.query.parse(lang, "(identifier) @ident")
+    local replacements = {}
+
+    if query then
+      for _, node in query:iter_captures(root, content) do
+        if vim.treesitter.get_node_text(node, content) == old_name then
+          local start_row, start_col, end_row, end_col = node:range()
```

Luacheck (1.2.0)

[warning] unused variable 'end_row'

(W211)